.github/workflows/nightly.yml

```diff
@@ -32,4 +32,4 @@ jobs:
       - name: Install cargo audit
         run: cargo install cargo-audit
       - name: Execute cargo audit
-        run: cargo audit
+        run: cargo audit --ignore RUSTSEC-2021-0139
```

Comment:
An alternative to this is to add this ID to the [existing](https://github.com/parallaxsecond/parsec-tool/blob/main/.cargo/audit.toml#L2) list of ignores 😄 Might also be worth checking whether the other ones have been resolved.

Reply:
Actually, maybe that's something to include in the release checklist? Don't remember if it's there

Reply:
Will add it to the checklist as well. :)